Source/WebCore/css/values/primitives/CSSPrimitiveNumericRange.h

```diff
@@ -114,6 +114,56 @@ template<Range range, std::floating_point T> constexpr T clampToRange(T value, T
     );
 }
 
+// Clamps an unsigned integral value to within `range`.
+template<Range range, std::unsigned_integral T> constexpr T clampToRange(T value)
+{
+    static_assert(range.min >= 0);
+
+    if constexpr (range.max == Range::infinity) {
+        return clampTo<T>(
+            value,
+            range.min,
+            std::numeric_limits<T>::max()
+        );
+    } else {
+        return clampTo<T>(
+            value,
+            range.min,
+            std::min<T>(range.max,  std::numeric_limits<T>::max())
```

Comment:
```suggestion
            std::min<T>(range.max, std::numeric_limits<T>::max())
```
Also, is there a guarantee this can be done at compile time?

Reply:
I wish c++ had some keyword/syntax to say "run this function only at compile time, erroring otherwise" like:

```c++
[[require_consteval]] std::min<T>(range.max, std::numeric_limits<T>::max())
```

but none exists. That said, these are all `constexpr` values so it will happen at compile time.